### PR TITLE
Feature - update ui cart

### PR DESCRIPTION
**Pull request by at-phatnguyen**

### CODE REVIEW

File: MyApp/View/Controllers/Cart/CartViewController.swift [hunk 1]
```diff
@@ -9,38 +9,69 @@
...
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return viewModel.numberOfItems(inSection: section)
     }
+
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = cartTableView.dequeueReusableCell(withIdentifier: "CartTableViewCell", for: indexPath)
+        let cell = cartTableView.dequeue(CartTableViewCell.self)
         return cell
     }
+
```
Comment (at-daiho) [inline MyApp/View/Controllers/Cart/CartViewController.swift]: WARNING - Unused parameter "action" in a closure should be replaced with _.

Comment (at-daiho) [inline MyApp/View/Controllers/Cart/CartViewController.swift]: WARNING - Unused parameter "indexPath" in a closure should be replaced with _.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Cell/CartCell/CartTableViewCell.swift [hunk 2]
```diff
@@ -14,21 +14,25 @@ class CartTableViewCell: UITableViewCell {
...
+    @IBOutlet weak var costLabel: UILabel!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var itemImageView: UIImageView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        setViewCell()
+        updateView()
+    }
+    @IBAction func subButtonTouchInSide(_ sender: UIButton) {
```
Comment (at-truongnguyen) [inline MyApp/View/Cell/CartCell/CartTableViewCell.swift]: TouchUp chứ em hè?

File: MyApp/View/Cell/CartCell/CartTableViewCell.swift [hunk 3]
```diff
@@ -14,21 +14,25 @@ class CartTableViewCell: UITableViewCell {
...
+        print("sub")
+    }
+
+    @IBAction func addButtonTouchInSide(_ sender: UIButton) {
```
Comment (at-truongnguyen) [inline MyApp/View/Cell/CartCell/CartTableViewCell.swift]: at-truongnguyen just now Member
TouchUp chứ em hè?

**Pull request merged**